# Goodness of Fit (GOF)

The $\chi^2$ test can be used in a rather novel way:

<center><b><span style='color: purple;'>We can test a probability model.</span></b></center></br>

The $\chi^2$ GOF allows us to compare observed data to a given probability model. Perhaps the best example is that of eye color.

## Example: Eye Color

A recent release from the American Academy of Opthamology gives the proportion of various eye colors in the population of the United States.

<table style="width:55%">
  <tr><th></th><th>Blue</th><th>Green</th><th>Hazel</th> <th>Light Brown</th><th>Dark Brown</th> 
  </tr>
  <tr><th>Proportion</th><td style="text-align: center;">32%</td><td style="text-align: center;">15%</td><td style="text-align: center;">12%</td><td style="text-align: center;">16%</td><td style="text-align: center;">25%</td>
  </tr>
</table>

At UNG, recent class surveys resulted in the following sample eye color distribution from students at the university:

In [13]:
obs <- c(68,41,30,51,60)
#colnames(obs) <- c('Blue', 'Green', 'Hazel', 'Light Brown', 'Dark Brown')
#obs
prob <- c(0.32, 0.15, 0.12, 0.16, 0.25)
prob

[1] 0.32 0.15 0.12 0.16 0.25

In [14]:
chisq.test(obs, p = prob)


	Chi-squared test for given probabilities

data:  obs
X-squared = 5.2517, df = 4, p-value = 0.2624


### Reporting Out

Given that $p = 0.2624 > 0.05 = \alpha$, we fail to reject the null. We have no evidence that the observed data on eye color from UNG students departs from the nationwide probability.